draft-trammell-rains-protocol.md

```diff
@@ -392,30 +413,66 @@ For efficiency's sake, information elements within a shard common to all
 assertions (zone, context, signature) within the shard may be omitted from the
 assertions themselves.
 
+A bloom filter is a space-efficient probabilistic data structure represented as
```

Comment:
We should call this something else in the spec: "bloom filter" is indeed how it is implemented, but what it actually provides is a "probabalistic negative proof".

Reply:
It is called now pshard (probabilistic shard)